### PR TITLE
[adobe_analytics] Analytics mapping for Android

### PR DESCRIPTION
**Pull request by jeremie-movify**

### CODE REVIEW

**Review by petermnt — APPROVED** [review]

File: packages/adobe_analytics/README.md [hunk 1]
```diff
@@ -21,7 +21,27 @@ Adobe SDKs setup reference : https://aep-sdks.gitbook.io/docs/getting-started/in
...
+// 1. Set ACPCore's log level
+MobileCore.setLogLevel(LoggingMode.VERBOSE) // Optional
+
+// 2. Configure ACPCore with your Adobe Environment ID
+MobileCore.setApplication(application)
+MobileCore.configureWithAppID("<Your Adobe Environment ID>")
+
+// 3. Register Analytics & Identity extensions 
+try {
+    Analytics.registerExtension()
```
Comment (timrijckaert) [inline packages/adobe_analytics/README.md:36]: This can crash? 😢

File: packages/adobe_analytics/android/build.gradle [hunk 2]
```diff
@@ -41,4 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.adobe.marketing.mobile:analytics:1.+"
```
Comment (timrijckaert) [inline packages/adobe_analytics/android/build.gradle]: I think it's better to work with fixed versions.

**Review by timrijckaert — CHANGES_REQUESTED** [review]

File: packages/adobe_analytics/README.md (hunk 1 above)
Comment (jeremie-movify) [reply packages/adobe_analytics/README.md:36]: From *Analytics.java* :
```
public static void registerExtension() throws InvalidInitException {
    Core core = MobileCore.getCore();
    if (core == null) {
      throw new InvalidInitException();
    } else {
      try {
        String mobileCoreVersion = MobileCore.extensionVersion();
        String analyticsVersion = "JAVA-1.2.2-" + mobileCoreVersion + "-" + "AN";
        analyticsCore = new AnalyticsCore(core.eventHub, analyticsVersion);
      } catch (Exception var3) {
        throw new InvalidInitException();
      }
    }
  }
```

File: packages/adobe_analytics/android/build.gradle (hunk 2 above)
Comment (jeremie-movify) [reply packages/adobe_analytics/android/build.gradle]: I agree ; I've updated the PR accordingly 👍

**Pull request merged**